GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -329,7 +330,7 @@ string GetFixCommitMessage()
 
                 CommitData data = _commitDataManager.CreateFromRevision(_revision, _children);
                 Validates.NotNull(_commitDataBodyRenderer);
-                return _commitDataBodyRenderer?.Render(data, showRevisionsAsLinks: false) ?? string.Empty;
+                return _commitDataBodyRenderer.Render(data, showRevisionsAsLinks: false);
```

Comment:
I am not 100% convinced that this is safe. The data loading in this control is async, and without any cancelations. So, it's quite possible to get in a state where data is being loaded, and at the same time a repo is being unloaded which sets these variables to null.

Reply:
You are right, I stopped in the middle of a right movement.